bin/run-db-upload.py

```diff
@@ -85,9 +85,11 @@ def main(args):
     prev_data = get_prev_db_data()
     new_data = get_db_data()
     if not force and prev_data and prev_data['checksum'] == new_data['checksum']:
-        print('No update necessary')
+        print('No update necessary\n')
```

Comment:
Do you need the extra newline? Print adds one by itself.

Reply:
Yeah. Don't need the trailing newline.

Reply:
I was thinking the extra new line would help make it a bit easier to parse:

```+ python manage.py update_pocketfeed --quiet
No update necessary
[2020-04-08 17:28:25.748025] Clock job update_upload_database@bedrock-dev-clock-5f947c7699-z4xlt: finished successfully
[2020-04-08 17:32:44.212303] Clock job update_upload_database@bedrock-dev-clock-5f947c7699-z4xlt: starting```

I think having an empty newline in that log would be helpful, between the start/stop messages in the long. Easier to scan through the logs and see the start/stop of each job.  Though it looks like I put it in the wrong place, so will update to be on the finished successfully message.

Reply:
I don't think that's a thing. Each line is a line in papertrail and they're all jumbled anyway. I don't see how an empty line is going to help in a log.